.github/actions/spelling/advice.md

```diff
@@ -2,25 +2,32 @@
+- _excludes.txt_: Perl-style regexes to ignore specific paths, files or extensions.
+- _expect.txt_: Dictionary file of arbitrary strings that aren't words, but otherwise valid and aren't spelling mistakes.
+- _patterns.txt_: Technical strings that aren't made up of words or word stems but follow a certain structure or pattern can be skipped using Perl-styled regexes. Some technical strings are already covered in Lombiq's version, such as hex color codes, Git commit hashes, GUIDs, and more.
 
-You can provide these files in your own repository, but they must be placed under the path _.github/actions/spelling_.
+You can provide these files in your own repository, but they must be placed under the path _.github/actions/spelling_. To ease maintaining dictionary files (and keep a consistent behavior), keep the entries sorted alphabetically.
 
-## Guidelines for adding new dictionary entries
+## When not to add entries to dictionary files
 
 Before adding an entry to one of the dictionary files, consider the following:
 
-1. When confronted with unrecognized words in a spell checking report, consider which of those are actually words that make sense to type, instead of just being remainders (because some parts of the original text were replaced with a space character due to matching an earlier dictionary entry) of another word or a technical string.
-2. Technical strings that aren't made up of words or word stems but follow a certain structure or pattern can be allowed with a regex in _patterns.txt_. Some technical strings are already covered, such as hex color codes, Git commit hashes, GUIDs, and more.
-3. Rare cases that we don't expect to show up overall more than 3 times and special cases that are valid in one place, but would otherwise be a typo should be ignored in-place without adding them to the dictionary. Placing the `#spell-check-ignore-line` string somewhere in a line (for example in a comment at the end of a line of code) will exclude that line completely from spell checking.
+1. When confronted with unrecognized words in a spell checking report, consider which of those are actually words that make sense to type, instead of just being remainders (because some parts of the original text were replaced with a space character due to matching an earlier entry) of another word or a technical string.
+2. Rare cases that we don't expect to show up overall more than 3 times and strings that are valid in a single situation should be ignored in-place without adding them to a dictionary file. Placing the `#spell-check-ignore-line` string somewhere in a line (for example in a comment at the end of a line of code) will exclude that line completely from spell checking.
+
+## Helper scripts for local development
+
+When you're using your own configuration files in conjuction with others (such as the ones from check-spelling or Lombiq's) these scripts might come on handy to reduce the number of entries you need to add to your own files:
```

Comment:
> (such as the ones from check-spelling or **Lombiq's**)

I think this could be made a bit clearer regarding what Lombiq resource can be used here.

> scripts might come **on** handy

Looks like a typo, the phrase is "come **in** handy".